application/classes/beans/vendor/expense/create.php

```diff
@@ -34,7 +34,7 @@
 @required lines ARRAY An array of objects representing line items for the expense.
 @required @attribute lines description STRING The text for the line item.
 @required @attribute lines amount DECIMAL The amount per unit.
-@required @attribute lines quantity INTEGER The number of units.
+@required @attribute lines quantity DECIMAL The number of units.
```

Comment:
And let's update these to be the same as the customer-side ones...  that's lagging documentation that should have been fixed a while ago.

```
@required @attribute lines quantity DECIMAL The number of units ( up to three decimal places ).
```